weed/mount/weedfs_file_mkrm.go

```diff
@@ -130,6 +130,11 @@ func (wfs *WFS) Unlink(cancel <-chan struct{}, header *fuse.InHeader, name strin
 		return code
 	}
 
+	// you can't delete a frozen file in worm mode
+	if wfs.option.WORM && entry != nil && entry.HardLinkCounter <= 1 && entry.Frozen {
```

Comment:
need to prevent hard linking to a writeOnceReadMany file/dir

Reply:
in here, wfs.option.WORM should not matter

Reply:
there is an example - a worm filesystem which support hardlink: https://docs.oracle.com/cd/E22586_01/html/E22571/gkxss.html